src/Processing.js

```diff
@@ -8062,6 +8071,13 @@
           j = i % PConstants.SINCOS_LENGTH;
           p.vertex(centerX + cosLUT[j] * hr, centerY + sinLUT[j] * vr);
         }
+
+        if (mode === PConstants.CHORD)
```

Comment:
same notes about OPEN here: if we add it to the conditional check, PConstants.OPEN will simply work, as the fill rules for CHORD and OPEN are identical.